nlp/2021_11_25_gpt3mix/nlp_gpt3mix.ipynb

```diff
@@ -0,0 +1,1966 @@
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Yu0JVPbQk1zB"
+      },
+      "source": [
+        "synthetic_gptj_10_ds = load_from_disk('./drive/MyDrive/text_augmentation/gpt-j/10')\n",
+        "synthetic_gptj_50_ds = load_from_disk('./drive/MyDrive/text_augmentation/gpt-j/50')\n",
+        "synthetic_gptj_100_ds = load_from_disk('./drive/MyDrive/text_augmentation/gpt-j/100')\n",
+        "synthetic_gptj_200_ds = load_from_disk('./drive/MyDrive/text_augmentation/gpt-j/200')"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "Mpq3vNSMFWx_",
+        "outputId": "e11f2b52-0d25-4684-b882-37fa1a0ea4bb"
+      },
+      "source": [
+        "print(\"Dataset of 10 synthetic samples:\")\n",
+        "print(\"Text: {} \\nLabel: {}\\n\".format(synthetic_gptj_10_ds['text'][5], idx2label[synthetic_gptj_10_ds['label'][5]]))\n",
+        "print(\"Dataset of 50 synthetic samples:\")\n",
+        "print(\"Text: {} \\nLabel: {}\\n\".format(synthetic_gptj_50_ds['text'][5], idx2label[synthetic_gptj_50_ds['label'][5]]))\n",
+        "print(\"Dataset of 100 synthetic samples:\")\n",
+        "print(\"Text: {} \\nLabel: {}\\n\".format(synthetic_gptj_100_ds['text'][5], idx2label[synthetic_gptj_100_ds['label'][5]]))\n",
+        "print(\"Dataset of 200 synthetic samples:\")\n",
+        "print(\"Text: {} \\nLabel: {}\\n\".format(synthetic_gptj_200_ds['text'][5], idx2label[synthetic_gptj_200_ds['label'][5]]))"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Dataset of 10 synthetic samples:\n",
+            "Text: he didn't even call me back  \n",
+            "Label: surprise\n",
+            "\n",
+            "Dataset of 50 synthetic samples:\n",
+            "Text: i am feeling angry and sad at the same time  \n",
+            "Label: anger\n",
+            "\n",
+            "Dataset of 100 synthetic samples:\n",
+            "Text: i have the feeling she was amused and delighted  \n",
+            "Label: joy\n",
+            "\n",
+            "Dataset of 200 synthetic samples:\n",
+            "Text: how dare you?  \n",
+            "Label: anger\n",
+            "\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ygrmMapNJrsg"
+      },
+      "source": [
+        "Even though GPT-J is much smaller than GPT-3, it manages to generate high quality samples 👏."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "_PaeBV_AWb_-"
+      },
+      "source": [
+        "### GPT-Neo\n",
+        "\n",
+        "A third alternative is GPT-Neo that is also released by Eleuther AI. It has 2.7 billion parameters and is also publicly available. \n",
```

Comment:
Bit of an 'off' sentence structure, better might be:
A third alternative is GPT-Neo, which was also made and released by Eleuther AI